.github/workflows/e2e.yml

```diff
@@ -0,0 +1,102 @@
+name: "Test: End to End"
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  end-to-end-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Workspace prefix
+        id: prefix
+        run: echo "::set-output name=name::action-e2e-test-$(date +%s)"
+      
+      - name: Workspace names
+        id: workspaces
+        env:
+          workspaces: '["alpha", "beta"]'
+        run: echo "::set-output name=names::${{ toJSON(env.workspaces) }}"
```

Comment:
Not sure `toJSON` and the indirection via `env` is needed, you should just be able to include the JSON string here. Use single quotes for `echo ''` and double in the JSON.

Reply:
yeah agreed, i just thought it was easier to read see

Reply:
Kind confusing, not really obvious that `env` will be available in this context. And then `env.workspaces` is a string, so calling `toJSON` should return the content wrapped in double quotes. Then the behavior depends on whether `::set-output name=foo::"bar"` actually results in quotes in the resulting output.